packages/fast-components-class-name-contracts-msft/src/carousel/index.ts

```diff
@@ -67,4 +67,14 @@ export interface CarouselClassNameContract {
      * The dark slide theme
      */
     carousel__themeDark?: string;
+
+    /**
+     * The previous slide transition
+     */
+    carousel__slideTransitionDirectionPrevious?: string;
```

Comment:
I haven't looked at how contracts are done closely so this comment may seem out of left field.  Can you describe how you came to this name?  Should there be another property for the type of animation/transition?  It's already a CarouselClass so why does Carousel need to be in the name of the properties as well.  Feels redundant.  Consider PreviousSlide or NextSlide.

Reply:
@awentzel 
All class name contracts include their root element. This is common across all MSFT components.

This is simply a class that a dev can key off of the add any animations or transitions when switching slides. The name was chosen to be explicit.

I came to this naming convention by this rationale: 
`[main element]_[element][action][action type]`
`[carousel]_[slide][transition][direction previous]`